jobs/deploy-fastly.groovy

```diff
@@ -163,6 +163,35 @@ def notifyWithVersionInfo(oldActive, newActive) {
    withSecrets.slackAlertlibOnly() {
       sh("echo ${exec.shellEscape(body)} | ${exec.shellEscapeList(cmd)}");
    }
+
+   // Let's tell #whats-happening as well, at least for prod deploys.
+   // But we don't alert for fastly-compute, since that is deployed
+   // automatically as part of the normal deploy process, and gets
+   // alerted on #whats-happening via that mechanism.
+   if (params.TARGET == "prod" && params.SERVICE != "khanacademy-org-compute") {
```

Comment:
I was initially thinking that we'd do it for just VCL. But doing it for all of them seems useful!